### DISCUSSION TITLE
Is there a port of EasyEffects for Android?

### DISCUSSION THREAD

**Discussion by Kevin-R-999** [wwmm/easyeffects#2433] [Q&A]
Even if greatly simplified with only EQ and compression it would be better than most EQ banner ad apps.

**Comment by wwmm** [comment]
There isn't and I have my doubts it is even possible. Everything is different on Android. The sound server, the approach to create the graphical interface, etc.

**Reply by Kevin-R-999** [reply]
Thanks for the reply. I was really impressed with the power of EasyEffects. Just hoping a less predatory EQ existed for Android.
